docs/03-tutorials/00-security/sec-02-setup-mtls-gateway.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Set up a mutual TLS Gateway 
+---
+
+This tutorial shows how to set up a mutual TLS Gateway and configure authentication based on certificate details.
+
+## Prerequisites
+
+To follow this tutorial, use Kyma 2.6 or higher.
+
+Before you start, Set up a [`custom-domain`](../00-api-exposure/apix-02-setup-custom-domain-for-workload.md) and prepare a certificate to expose a workload.
+
+## Steps
+
+1. Export the following values as environment variables:
+
+   ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME} 
+   ```
+   >**NOTE:** `DOMAIN_NAME` is the domain that you own, for example, api.mydomain.com
+
+2. Create a Namespace and export its value as an environment variable. Run:
+   >**NOTE:** Skip this step if you already have a Namespace
+
+      ```bash
+   export NAMESPACE={NAMESPACE_NAME}
+   kubectl create ns $NAMESPACE
+   kubectl label namespace $NAMESPACE istio-injection=enabled --overwrite
+   ```
+
+3. Create Client Root CA and CLient certificate signed by Client Root CA. Export the following values as environment variables and run the command provided:  
+      ```bash
+    export CLIENT_ROOT_CA_KEY_FILE=client-root-ca.key
```

Comment:
which of these exports can be re-used in expose and secure workload? i would like to avoid duplication